### ISSUE TITLE
no-unused-vars with varsIgnorePattern always triggering

### ISSUE THREAD

**Issue by tivac** [ota-meshi/eslint-plugin-svelte#154]
We have a few places in our codebase where we need to iterate an array-like object (`{ length : 5 }`) and we don't care about the contents of each index, because there's nothing there. Maybe we just need to draw something 5 times, or need to use the index value to grab something from somewhere else.

The simplest possible version of that looks like this:

```svelte
{#each { length : 5 } as _}
    <div>thing</div>
{/each}
```

I've tried to account for using `_` as the iteration value I don't care about using `no-unused-vars`' support for `varsIgnorePattern` but nothing I've tried has seemed to work, If I use this package I get `defined but never used` results.

```bash
> eslint .

./eslint-plugin-svelte-unused-vars/component.svelte
  1:26  warning  '_' is defined but never used  no-unused-vars

✖ 1 problem (0 errors, 1 warning)
```

Here's a repro as a gist:

https://gist.github.com/tivac/7b0623dfd128d0a60c8ce05069f79e86

Is there a different way I should be trying to set this up? I can work around it for now by using `<!-- eslint-disable no-unused-vars --> above each `{#each}` tag, but it's clunkier than I'd like.

**Comment by ota-meshi** [comment]
Thank you for positing issue!
Variables assigned by `{#each}` are considered parameter variables (for compatibility).
Therefore, you should use `argsIgnorePattern` instead of `varsIgnorePattern`.

https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJyFj7EKg0AQRH9l2XSiHATSHId9uvTZRA5dVJC9sHfayP17FCGQJumGgfeGWbENHaNFF1sdX6kmMQXHaZQEEqpZ5shdtXiNFu6ErBqUsIQVCL328dpLUL75lFiF0G71syGE/CgMycQJDgWJM58FkvXEvh02y8TSpwEsXCCDj9BkEgDXjUudhlF6Z/a4AWYH8g/y2PmHY4k6TxzRrvj9D+055ze/5F3r

**Comment by tivac** [comment]
Thanks @ota-meshi, finally had a chance to test that out today and it worked perfectly 🙏🏻
